lib/Arduino_Core_Buddy/CMakeLists.txt

```diff
@@ -21,13 +21,8 @@ target_include_directories(
   )
 
 target_compile_definitions(
-  CoreBuddy
-  PUBLIC
-  ARDUINO=187
-  $<$<STREQUAL:${ARDUINO_CORE_TARGET},STM32F4xx>:STM32F4xx>
-  HAVE_HWSERIAL2
-  USBD_USE_CDC
-  USBCON
+  CoreBuddy PUBLIC ARDUINO=187 $<$<STREQUAL:${ARDUINO_CORE_TARGET},STM32F4xx>:STM32F4xx>
```

Comment:
do we really want the compile definitions squashed into 2 lines?

Reply:
Not ideal here, I agree. But I tried a few more settings and when they improved the situation here, they broke the formatting on much more other places. So I would go with this for now.